server/graphql/v2/query/RequiredBankInformationQuery.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import RequiredBankInformation from '../object/RequiredBankInformation';
+import models from '../../../models';
+
+const RequiredBankInformationQuery = {
+  type: RequiredBankInformation,
+  args: {
+    slug: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'The public id identifying the host collective',
+    },
+  },
+  async resolve(_, args, req): Promise<any> {
```

Comment:
I don't see the benefit of Typescript in this file.

Should we create .ts files even when we're not really using it?

Reply:
I think it is a good idea because you get static checking from other imported TS.
I don't see a lot of value in requiring us to be explicit about the returning value though.
It should provide more value when we migrate all models to TS.

Reply:
I agree with that. Even if you're not actively typing everything, that doesn't mean you're not using typescript. For example:

```es6
const x = 42;
const y = 'test';
const z = x / y; // typescript will not be happy
```

On the other hand, I don't see any downside about using ts files